languages.toml

```
@@ -1380,7 +1380,8 @@ source = { git = "https://github.com/mtoohey31/tree-sitter-gitattributes", rev =
 name = "git-ignore"
 scope = "source.gitignore"
 roots = []
-file-types = [".gitignore", ".gitignore_global"]
+file-types = [".gitignore", ".gitignore_global", ".ignore", "prettierignore",
```

```suggestion
file-types = [".gitignore", ".gitignore_global", ".ignore", ".prettierignore",
```

It looks like this wasn't resolved.

Also, no need to hard-wrap the line, we don't enforce a maximum line length for languages.toml

---

Apologis, i thought the suggested change was the wrapping and didn't spot the missing dot. I will fix that when im back on tomorrow.